gpgme.go

```diff
@@ -389,7 +390,11 @@ func (c *Context) GetKey(fingerprint string, secret bool) (*Key, error) {
 	key := newKey()
 	cfpr := C.CString(fingerprint)
 	defer C.free(unsafe.Pointer(cfpr))
-	return key, handleError(C.gpgme_get_key(c.ctx, cfpr, &key.k, cbool(secret)))
+	err := handleError(C.gpgme_get_key(c.ctx, cfpr, &key.k, cbool(secret)))
+	if key.k == nil {
```

Comment:
I want to also note that is go idiomatic to return an error stating that the key isn't found rather than returning `nil, nil` or checking for key nilness in library implementors... If @mtrmac I can define a precise error for this scenario and library implementors can choose to ignore a not found error (similar to what `os.Stat` does in the golang standard library when a file is not found`)